setup.py

```diff
@@ -40,7 +40,7 @@ def check_for_openmp():
         cc = os.environ['CC']
     except KeyError:
         cc = 'gcc'
-    with open(filename, 'w', 0) as file:
+    with open(filename, 'w', 1) as file:
```

Comment:
this is for the optional argument `buffering` in the function `open`. `0` means unbuffered which causes problems in Python 3, and `1` means line buffered. A negative value here say `-1` means system default settings.